tests/ecosystem/upgrade/test_resources.py

```diff
@@ -90,3 +98,22 @@ def test_pod_io(
         helpers.wait_for_resource_state(pod, constants.STATUS_RUNNING, timeout=600)
         job_name = pod.get_labels().get("job-name")
         job_obj.delete(resource_name=job_name)
+
+
+@post_upgrade
+@bugzilla("1974343")
+@skipif_ocs_version("<4.4")
+def test_pod_log_after_upgrade():
+    """
+    Check OSD/MON/MGR pod logs after upgrade and verify the expected log exist
+
+    """
+    osd_pod_objs = get_osd_pods() + get_mon_pods() + get_mgr_pods()
+    pod_names = [osd_pod_obj.name for osd_pod_obj in osd_pod_objs]
+    expected_log_after_upgrade = "set uid:gid to 167:167 (ceph:ceph)"
```

Comment:
This specific log pattern will be logged in all these pods frequently? 
Please add a comment explaining about this log pattern above this line.

Reply:
done